Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -390,6 +406,10 @@ void CGPathAddArc(CGMutablePathRef path,
  @Status Interoperable
 */
 void CGPathMoveToPoint(CGMutablePathRef path, const CGAffineTransform* m, float x, float y) {
+    if (path == NULL) {
```

Comment:
I didn't find a RETURN_IF_NULL macro available in the project.  I only found the following matching NULL macros which aren't appropriate for these cases as we aren't returning HRESULT:

```
  C:\WinObjC\Frameworks\include\wil\result.h(823):#define RETURN_IF_NULL_ALLOC(ptr)                  \
  C:\WinObjC\Frameworks\include\wil\result.h(904):#define RETURN_IF_NULL_ALLOC_LOG(ptr)                  \
  C:\WinObjC\Frameworks\include\wil\result.h(985):#define RETURN_IF_NULL_ALLOC_MSG(ptr, fmt, ...)                          \
  C:\WinObjC\Frameworks\include\wil\result.h(1061):#define RETURN_IF_NULL_ALLOC_EXPECTED(ptr) \
  Matching lines: 4    Matching files: 1    Total files searched: 4039
```